Zend/zend_builtin_functions.c

```diff
@@ -140,9 +140,13 @@ ZEND_FUNCTION(gc_status)
 
 	array_init_size(return_value, 3);
 
+	add_assoc_bool_ex(return_value, "active", sizeof("active")-1, status.active);
```

Comment:
`active` can be misunderstood with `enabled`, `running` name can be better

can user obverse this to be true somewhere? In destructor?

Reply:
From my point of view, it can be called whatever, but @dstogov named the other fields after the associated GC_G() global and maybe he has an opinion on whether that convention should continue.

Yes, `active` can be true in a destructor. The bug I linked to (#9323) occurs when a destructor is called from the garbage collector.

Reply:
I changed it to running.